baselines/NER/docile_train_NER_multilabel_layoutLMv3.py

```diff
@@ -842,12 +842,24 @@ def prepare_hf_dataset(
         default=1,
         help="",
     )
+    parser.add_argument("--resume", action="store_true")
     args = parser.parse_args()
 
     print(f"{datetime.now()} Started.")
 
     show_summary(args, __file__)
 
+    output_dir_exists = os.path.exists(args.output_dir) and os.listdir(args.output_dir) != [
```

Comment:
This is duplicated, but it is probably not the only such part, so I guess we can keep it

Reply:
Unfortunately most of the LayoutLMv3 and RoBERTa training and inference code are duplicated, it would be non-trivial to try and deduplicate it now.